HomeWork1/task26.py

```diff
@@ -0,0 +1,24 @@
+# Задача 26:  Напишите программу, которая на вход принимает два числа A и B, и
+# возводит число А в целую степень B с помощью рекурсии.
+
+# *Пример:*
+
+# A = 3; B = 5 -> 243 (3⁵)
+#     A = 2; B = 3 -> 8 
+
+# Задача 26:  Напишите программу, которая на вход принимает два числа A и B,
+# и возводит число А в целую степень B с помощью рекурсии.
+
+# *Пример:*
+
+# A = 3; B = 5 -> 243 (3⁵)
+#     A = 2; B = 3 -> 8
+
+a = int(input("Введите число: "))
+b = int(input("Введите в какую степень его возвести: "))
+def exponentiate(a, b):
+    if b == 1:
+        return a**b
```

Comment:
???
Если показатель степени будет 1, то степень равна
```suggestion
        return a
```